### ISSUE TITLE
why images are downloaded to a ziped file using mozilla firefox ? this is not good

### ISSUE THREAD

**Issue by conteudoanimal** [furyutei/twMediaDownloader#112]
Hello,

When I use this extension on mozilla firefox the images are downloaded to a zipped file ... 
Why it is not download just the file like it is when I download a video ?

We need to open the zip file and extract the image from it ...

As it is now ... it is better/easier  mouse right click and download it raw ...

thanks in advance !

Marcos
www.conteudoanimal.com.br

**Comment by estos73** [comment]
I agree for single images. When you have a single image in a Twitter post and click the img button, you end up with the image in a zip file on your computer. That is very annoying. I totally want zip files if there are more than 1 image in a post.

maybe there could be a setting in Options for the img button in individual posts to switch between preferred modes:
single img download [] zip [x] raw

単一の画像については同意します。 Twitter の投稿に 1 つの画像があり、[画像] ボタンをクリックすると、その画像が zip ファイルとしてコンピュータに保存されます。 それはとても迷惑です。 投稿に複数の画像がある場合は、完全に zip ファイルが必要です。

おそらく、個々の投稿の img ボタンのオプションに、優先モードを切り替えるための設定がある可能性があります。
シングル img ダウンロード [] zip [x] raw

ご清聴ありがとうございました
